admin_honeypot/models.py

```diff
@@ -1,15 +1,24 @@
+import ast
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 from admin_honeypot import listeners
 
 
 class LoginAttempt(models.Model):
-    username = models.CharField(_("username"), max_length=255, blank=True, null=True)
-    ip_address = models.GenericIPAddressField(_("ip address"), protocol='both', blank=True, null=True)
-    session_key = models.CharField(_("session key"), max_length=50, blank=True, null=True)
-    user_agent = models.TextField(_("user-agent"), blank=True, null=True)
-    timestamp = models.DateTimeField(_("timestamp"), auto_now_add=True)
-    path = models.TextField(_("path"), blank=True, null=True)
+    username = models.CharField(
```

Comment:
You shouldn't include whitespace changes in this pull request. Makes it much harder for @dmpayton to review. For reference, whitespace changes should be in its own pull request.